detail.scss

```diff
@@ -0,0 +1,51 @@
+.title {
```

Comment:
Check the bottom padding for this class so you can place the border in a better position 😄

Reply:
Done